### ISSUE TITLE
Why did system restart improve model behaviour?

### ISSUE THREAD

**Issue by jetjodh** [tensorflow/tensorflow#31471]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows10
- TensorFlow installed from (source or binary):binary
- TensorFlow version (use command below):1.13.1
- Python version: 3.7
- CUDA/cuDNN version: 10
- GPU model and memory: 105Ti(mobile) 4GB

**Describe the current behavior**
I was trying to train custom models and pretrained models on a custom dataset but no models were learning anything. The accuracy after many epochs was always stuck at 55-60% accuracy. 
After many days of frustration, I shut down the system. The next day the system booted and the models were learning and converging near 100%. How and why did this behavior occur?

**Comment by ymodak** [comment]
I apologize, but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new). Please provide all the information it asks. Thank you.

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
